src/content/blog/2024-08-22-the-ugrc-api-address-location-lane.mdx

```diff
@@ -0,0 +1,45 @@
+  The UGRC API is a mature service that provides address locating. We are very proud to offer this service for free, but it's not the only solution for every addressing problem. Read this to understand the limitations and capabilities of our geocoding endpoint.
+date: '2024-08-22T23:47:42.633Z'
+category: API
+cover_image: /src/images/pillar-blog/default-social-card.png
+cover_image_alt: ugrc social card
+---
+
+import centerlineImage from '@images/blog/2024-08-22-the-ugrc-api-address-location-lane/centerline-address-match-example.png';
+import { Image } from 'astro:assets';
+
+The [UGRC API](https://api.mapserv.utah.gov/) is a valuable resource for [address locating](https://api.mapserv.utah.gov/docs/v1/endpoints/geocoding/). We are proud to offer this service to you for free, however it's not the only solution for your addressing questions. Let's explore the layers that enable the address locating `/geocode` endpoint to help you understand what this service offers and its limitations.
+
+## The foundational data
+
+The UGRC provides authoritative GIS data for Utah through the [SGID](/products/sgid/sgid-index/). By aggregating data from local governments and standardizing it, we are able to offer this comprehensive statewide resource. Our address locating endpoint provides excellent address match rates because of the collaborative efforts of our local address authority partners and their dedicated work. Together, we maintain two foundational datasets that power the UGRC API -- [address points](/products/sgid/location/address-points/) and [road centerlines](/products/sgid/transportation/road-centerlines/). The [zip codes](/products/sgid/boundaries/zip-codes/) and [address systems](/products/sgid/location/address-system-quadrants/) participate and are worth mentioning but we won't be discussing them further.
+
+The road centerlines and address point data is regularly updated, with the "Big 5" counties (Salt Lake, Utah, Davis, Weber, Washington) providing monthly updates. While the other counties provide updates on a less frequent schedule, the "Big 5" cover approximately 80% of Utah's population, ensuring great coverage in densely populated areas. Stay informed about the latest updates by following the [SGID Update](/blog/categories/sgid-updates/) blog category.
+
+## The data limitations
+
+While the UGRC API address locating endpoint is a powerful service, it's important to be aware of its limitations. The accuracy of our data depends on the quality and timeliness of the data provided by our address authority partners. In rural counties, where resources and funding may be limited, data gaps exist. This is most pronounced in counties like Juab, Piute, Uintah, and Sevier. We are working hard to improve the data quality, especially in these areas, but it's a work in progress. These data characteristics make it problematic to solely use our API for address validation for mail delivery, 911 response, or voter registration types of questions.
```

Comment:
@ZachBeck and @MichaelAGRC could you confirm that those are the rural counties where most of our data gaps exist?

Reply:
I'd add Daggett, Duchesne and Rich to that list.

Reply:
Wow that's like 25 percent of the counties but probably a very small population of addresses.

Reply:
Sevier and Piute shouldn't be on the list. It should be Daggett, Duchesne, Juab, Rich, Sanpete, Uintah, and maybe San Juan